WalletWasabi.Fluent/ViewModels/Wallets/Advanced/WalletCoins/WalletCoinsViewModel.cs

```diff
@@ -50,7 +50,7 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 			.AsObservableCache()
 			.Connect()
 			.TransformWithInlineUpdate(x => new WalletCoinViewModel(x))
-			.Replay()
+			.Replay(1)
```

Comment:
Same in L45?

Reply:
Good question. Let me check.

Reply:
Fixed. Now it's optimized as expected. Before it applied Replay twice and that wasn't a good idea. At least, we don't need that. Thanks for the observation!